crates/typst-library/src/math/frac.rs

```diff
@@ -1,9 +1,17 @@
 use typst_syntax::Spanned;
 
 use crate::diag::bail;
-use crate::foundations::{Content, Value, elem};
+use crate::foundations::{Cast, Content, Value, elem};
 use crate::math::Mathy;
 
+/// Fraction style
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Hash, Cast)]
+pub enum FracStyle {
+    Vertical,
+    Skewed,
+    Horizontal,
+}
+
```

Comment:
```suggestion
/// Fraction style
#[derive(Debug, Default, Copy, Clone, Eq, PartialEq, Hash, Cast)]
pub enum FracStyle {
    /// Stacked numerator and denominator with a bar.
    #[default]
    Vertical,
    /// Numerator and denominator separated by a slash.
    Skewed,
    /// Numerator and denominator placed inline and parentheses are not absorbed.
    Horizontal,
}

```
Adding the docs here should make sure it ends up like https://typst.app/docs/reference/model/ref/#parameters-form in the docs.